index.html

```diff
@@ -3,6 +3,8 @@
   <head>
     <link rel="stylesheet" href="style.css">
     <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <main id="main">
 <h1 id="title">Dr. Norman Borlaug</h1>
 <p>The man who saved a billion lives</p>
```

Comment:
It's better to increase the font size of the p elements to make it more readable and improve the user-experience.